.github/codecov.yml

```diff
@@ -1,2 +1,15 @@
 comment:
-  layout: 'reach,diff,footer'
+  layout: "reach, diff, flags, files"
+  show_carryforward_flags: true
+
+flags:
+  clientmonorepo:
+    carryforward: true
+    paths:
+      - client/
+
+  # PHP backend and legacy frontend
+  unittests:
+    carryforward: true
+    paths:
+      - web/
```

Comment:
Just noting PHP tests will always be in `web/tests/php`, there is also a `web/tests/js` in there.

Reply:
Codecov cares about what files change - not about what tests are ran.